### DISCUSSION TITLE
Error while trying to register artifacts using Maven Plugin

### DISCUSSION THREAD

**Discussion by lekhamaru** [Apicurio/apicurio-registry#5486] [Q&A]
Hello

While trying to register an artifact, I used the example given at the following location. 
https://github.com/Apicurio/apicurio-registry/blob/main/examples/avro-maven-with-references-auto/pom.xml

I am getting the following error   - [ERROR] Exception while registering artifact [avro-maven-with-references] / [TradeKey]
com.microsoft.kiota.ApiException: the server returned an unexpected status code and no error class is registered for this code 404

 Failed to execute goal io.apicurio:apicurio-registry-maven-plugin:3.0.1:register (register-artifact) on project apicurio-registry-examples-avro-maven-with-references: Errors while registering artifacts ... -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.apicurio:apicurio-registry-maven-plugin:3.0.1:register (register-artifact) on project apicurio-registry-examples-avro-maven-with-references: Errors while registering artifacts ...

Versions being used are as follows

apicurio-registry-maven-plugin - 3.0.1
Apache Maven 3.9.9 (8e8579a9e76f7d015ee5ec7bfcdc97d260186937)
Maven home: C:\apache-maven-3.9.9
Java version: 18.0.2, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-18.0.2
Default locale: en_IN, platform encoding: UTF-8
OS name: "windows 11", version: "10.0", arch: "amd64", family: "windows"

Has anyone encountered anything similar.

**Comment by EricWittmann** [comment]
What version of the server are you using? How are you running it?

**Reply by lekhamaru** [reply]
Using the maven plugin running the command mvn package.
Using the docker image [apicurio/apicurio-registry-mem:latest-release]
